### DISCUSSION TITLE
ERROR:  cohortextractor must have a version specified (e.g. cohortextractor:0.5.2) (type=value_error)

### DISCUSSION THREAD

**Discussion by p-stehlik** [opensafely/documentation#1322] [Q&A] [answered]
I'm not really sure what to do about this error?

I am trying to generate multiple cohorts to look at participants over 1-month periods.

My project yaml looks like this:

`version: '3.3'

expectations:
  population_size: 1000

actions:

  generate_study_population:
    run: cohortextractor generate_cohort --index-date-range "2020-12-01 to 2021-12-01 by month" --output-format=csv.gz
    outputs:
      highly_sensitive:
        cohort: output/input_*.csv.gz
`

Just getting used to the features and exploring at the moment - appreciate the help.

**Comment by p-stehlik** [comment, marked as answer]
Realised the mistake

run: cohortextractor generate_cohort --index-date-range "2020-12-01 to 2021-12-01 by month" --output-format=csv.gz

should be:

run: cohortextractor:latest generate_cohort --index-date-range "2020-12-01 to 2021-12-01 by month" --output-format=csv.gz


FYI the original code I had was copied over from https://docs.opensafely.org/measures/ so this might need updating. Thanks :)

**Comment by benbc** [comment]
@p-stehlik: yes, that's absolutely right. Sorry about the error in the docs -- thanks for reporting it.

**Comment by benbc** [comment]
Docs fix here: https://github.com/opensafely/documentation/pull/1324.
